### PR TITLE
Download csv bug fix

### PR DESCRIPTION
**Pull request by jennya-alexeyeva**
## Why

Fix [bug](https://trello.com/c/bRiD4ZhL/8-investigate-debug-download-as-csv-action) where users are unable to download .csv files

## This PR
Return result of promise instead of just the promise itself

## Screenshots

image

## Verification Steps

Successfully downloaded a CSV file from the FormHistory page for a school with and without a library

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -22750,6 +22750,7 @@
       "version": "1.0.2",
       "resolved": "https://registry.npmjs.org/w3c-hr-time/-/w3c-hr-time-1.0.2.tgz",
       "integrity": "sha512-z8P5DvDNjKDoFIHK7q8r8lackT6l+jo/Ye3HOle7l9nICP9lf1Ci25fy9vHd0JOWewkIFzXIEig3TdKT7JQ5fQ==",
+      "deprecated": "Use your platform's native performance.now() and performance.timeOrigin.",
```
Comment (jackielincroft) [inline package-lock.json:22753]: do you know if this was automatically added by something? and do you need it in order for checks to pass and stuff? if not you might want to remove it (but if you need it to pass linting checks or something that's fine)

**Review by jackielincroft — APPROVED** [review]
sweet, this looks good! thanks for investigating this, glad the fix was simple!

File: package-lock.json (hunk 1 above)
Comment (jennya-alexeyeva) [reply package-lock.json:22753]: i'm not sure, i ran npm install and it must have accidentally been added from that. I'll try removing it and pushing again

Comment (jennya-alexeyeva) [reply package-lock.json]: @jackielincroft updated!

**Review by nate-kirschner — APPROVED** [review]
LGTM

**Pull request merged**